cantools/database/can/formats/dbc.py

```diff
@@ -1639,7 +1639,14 @@ def get_protocol(frame_id_dbc):
             except (KeyError, TypeError):
                 frame_format = None
 
-        if frame_format == 'J1939PG':
+        return frame_format
+
+    def get_protocol(frame_id_dbc):
```

Comment:
```suggestion
    def get_protocol(frame_format_dbc):
```

(in my mind, frame id is more or less synonymous with the arbitration ID...)